resources/views/people/debt/index.blade.php

```diff
@@ -46,7 +46,7 @@
           @endif
         </div>
         <div class="table-cell list-actions">
-          <a href="{{ route('people.debt.edit', ['people' => $contact->id, 'debtId' => $debt->id]) }}">
+          <a href="{{ route('people.debt.edit', [$contact, $debt]) }}">
```

Comment:
Why not keep the named attributes here ?

Reply:
@asbiin I would have to test it to be sure but it's one of two reasons. Either there was some quirk in laravels route binding that required it, or I got carried away with find and replace. Looking at I would  lean towards the first one.

Reply:
@turtles2 indeed ! The previous version was obviously buggy. Thanks !